cxf/pom.xml

```diff
@@ -57,8 +57,8 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
Er dette tilstrekkelig for cxf-lib? 
Må man ikke oppdatere cxf til 4.x, og fjerne div javax-avhengigheter?